### PR TITLE
Toolbar export action modal

### PR DESCRIPTION
**Pull request by thecalcc**

### CODE REVIEW

File: scripts/apps/authoring-react/toolbar/export-modal.tsx [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+            return httpRequestJsonLocal({
+                method: 'POST',
+                path: '/export',
+                payload: {
+                    item_ids: [this.props.article._id],
+                    validate: this.state.validate,
+                    format_type: this.state.selectedFormatter,
+                },
+            }).then((response: any) => {
+                return window.open(response.url);
```
Comment (petrjasek) [inline scripts/apps/authoring-react/toolbar/export-modal.tsx:66]: we might also close the modal once the download is triggered

File: scripts/apps/authoring-react/toolbar/export-modal.tsx [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+
+        return (
+            <Modal
+                size="medium"
+                onHide={this.props.closeModal}
+                visible
+                zIndex={1050}
+                headerTemplate={gettext('Export')}
+            >
+                <Spacer v gap="32">
```
Comment (petrjasek) [inline scripts/apps/authoring-react/toolbar/export-modal.tsx:86]: would be good to use some constants for such numbers like 32 so it's consistent and defined in 1 place

Comment (petrjasek) [reply scripts/apps/authoring-react/toolbar/export-modal.tsx:87]: I see

**Comment by tomaskikutis** [comment]
reviewed

**Pull request merged**